static/app/components/profiling/flamegraph/continuousFlamegraph.tsx

```diff
@@ -97,10 +98,27 @@ function collectAllSpanEntriesFromTransaction(
   return allSpans;
 }
 
-function getMaxConfigSpace(profileGroup: ContinuousProfileGroup): Rect {
-  // We have a transaction, so we should do our best to align the profile
-  // with the transaction's timeline.
+function getMaxConfigSpace(
+  profileGroup: ContinuousProfileGroup,
+  transaction: EventTransaction | null,
```

Comment:
I think this is fine for now because we're aiming for feature parity with the transaction profiles but there's a world where we won't have a transaction for a continuous profile so we should rely on the start/end timestamps passed to determine the config space instead of the transaction.

Reply:
Yup, I realized that. It'll need some more tweaking so I'm punting it till we get to adapt this to work without perf